dev-libs/libdnet/libdnet-1.14-r2.ebuild

```diff
@@ -5,29 +5,35 @@ EAPI=7
 AT_M4DIR="config"
 PYTHON_COMPAT=( python3_{6,7,8} )
 DISTUTILS_OPTIONAL=1
-inherit autotools distutils-r1 git-r3
+inherit autotools distutils-r1
 
 DESCRIPTION="simplified, portable interface to several low-level networking routines"
 HOMEPAGE="https://github.com/ofalk/libdnet"
-EGIT_REPO_URI="https://github.com/ofalk/libdnet"
-EGIT_BRANCH="devel"
+SRC_URI="https://github.com/ofalk/${PN}/archive/${P}.tar.gz"
 LICENSE="LGPL-2"
 
 SLOT="0"
-KEYWORDS=""
-IUSE="python static-libs test"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+IUSE="python test"
 
 DEPEND="
 	python? ( ${PYTHON_DEPS} )
 "
 RDEPEND="
 	${DEPEND}
 "
+BDEPEND="
+	python? (
+		dev-python/cython
+	)
```

Comment:
Hello,
I think you need a `dev-python/cython[${PYTHON_USEDEP}]` or `dev-python/cython[${PYTHON_MULTI_USEDEP}]` here.